spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -45,14 +45,6 @@
     then_I_see_the_new_step_by_step_page
   end
 
-  scenario "User deletes a step by step page", js: true do
-    given_there_is_a_step_by_step_page_with_steps
-    and_I_visit_the_index_page
-    and_I_delete_the_step_by_step_page
```

Comment:
Can you also delete the function that this calls as it's no longer being used.

Reply:
Done 👍